packages/api/src/routes/medical/patient.ts

```diff
@@ -440,15 +440,17 @@ async function checkResourceLimit(incomingAmount: number, patient: Patient) {
     });
     if (currentAmount + incomingAmount > MAX_RESOURCE_STORED_LIMIT) {
       throw new BadRequestError(
-        `Reached maximum number of resources per patient in Sandbox mode ` +
-          `(current: ${currentAmount}, incoming: ${incomingAmount}, max: ${MAX_RESOURCE_STORED_LIMIT})`
+        `Reached maximum number of ${MAX_RESOURCE_STORED_LIMIT} resources per patient in Sandbox mode `,
+        null,
+        { currentAmount, incomingAmount, MAX_RESOURCE_STORED_LIMIT }
       );
     }
     // Limit the amount of resources that can be created at once
     if (incomingAmount > MAX_RESOURCE_COUNT_PER_REQUEST) {
       throw new BadRequestError(
-        `Cannot create more than ${MAX_RESOURCE_COUNT_PER_REQUEST} resources at a time ` +
-          `(incoming: ${incomingAmount})`
+        `Cannot create more than ${MAX_RESOURCE_COUNT_PER_REQUEST} resources at a time.`,
+        null,
+        { incomingAmount }
```

Comment:
Let's keep messages sent to Sentry as literals, no dynamic values; it helps Sentry to group them together - [reference](https://www.notion.so/metriport/Development-Best-Practices-bb3574d3c83d46dc9914a9e5c5542ca7?pvs=4#291429931daf45a18db66d86df845749)

Reply:
But this isn't a message to Sentry. Or am I missing something?

Reply:
It might get sent to Sentry when it's processed by some code upstream. Actually, any 4xx doesn't get sent to Sentry [anymore](https://github.com/metriport/metriport/blob/565c9e06fc2a0a66e093949962b979a8694d11e5/packages/api/src/routes/helpers/default-error-handler.ts#L56), but (1) its about the pattern so we don't need to think about whether its a 4xx or 5xx, and (2) we might change that logic in the future.

I did notice looking at the code that we don't send `additionalData` by default as part of the API's response, we only use it to report to Sentry. Maybe we should, and then we can send more details to the caller w/o breaking the pattern to throw errors w/ static messages?

I'd move it to `additionalData` and create a ticket to expose it to the caller.

Reply:
Fixed. Issue ticketed [here](https://github.com/metriport/metriport/issues/2082).